prime-dotnet-webapi/Services/EnrolleeAgreementService.cs

```diff
@@ -120,6 +126,7 @@ public async Task CreateEnrolleeAgreementAsync(int enrolleeId)
         public async Task<Agreement> GetCurrentAgreementAsync(int enrolleeId)
         {
             return await _context.Agreements
+                .Include(a => a.AgreementVersion)
```

Comment:
There is a small issue here: the AgreementVersion has the text of the TOA on it; loading the text for every call to this method is kinda wasteful, especially considering no caller of this method is using the TOA text.